src/Stepper/README.md

```diff
@@ -0,0 +1,156 @@
+- StepperActionRow
+- StepperHeader
+- StepperHeaderStep
+categories:
+- Navigation
+- Content
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+
+---
+
+### Basic Usage
+
+A `Stepper` must wrap a set of composed subcomponents:
+- `Stepper.Header`
+- `Stepper.Step`
+- `Stepper.ActionRow`
+
+The order of steps is dictated by the order of `Stepper.Step` components in the code.
+
+ `Stepper.Step` and `Stepper.ActionRow` are hidden until their `eventKey` props match the `activeKey` on `Stepper`.
+
+```jsx live
+() => {
+  const steps = ['welcome', 'choose-cats', 'review'];
+  const [currentStep, setCurrentStep] = useState(steps[0]);
+
+  return (
+    <Stepper activeKey={currentStep}>
+      <Stepper.Header />
+
+      <Container size="sm" className="py-5">
+        <Stepper.Step eventKey="welcome" title="Welcome">
+          <h2>Welcome to the cat workflow</h2>
+          <HipsterIpsum numParagraphs={2} />
+        </Stepper.Step>
+
+        <Stepper.Step eventKey="choose-cats" title="Chooseth a cat">
+          <h2>Choose a cat!</h2>
+          <HipsterIpsum numParagraphs={2} />
+        </Stepper.Step>
+
+        <Stepper.Step eventKey="review" title="Review">
+          <h2>Review your work</h2>
+          <HipsterIpsum numParagraphs={2} />
+        </Stepper.Step>
+      </Container>
+
+      <div className="py-3">
+        <Stepper.ActionRow eventKey="welcome">
+          <Button variant="outline-primary" onClick={() => alert('Cancel')}>
+            Cancel
+          </Button>
+          <Stepper.ActionRow.Spacer />
+          <Button onClick={() => setCurrentStep('choose-cats')}>Next</Button>
+        </Stepper.ActionRow>
+
+        <Stepper.ActionRow eventKey="choose-cats">
+          <Button variant="outline-primary" onClick={() => setCurrentStep('welcome')}>
+            Previous
+          </Button>
+          <Stepper.ActionRow.Spacer />
+          <Button onClick={() => setCurrentStep('review')}>Next</Button>
+        </Stepper.ActionRow>
+
+        <Stepper.ActionRow eventKey="review">
+          <Button variant="outline-primary" onClick={() => setCurrentStep('choose-cats')}>
+            Previous
+          </Button>
+          <Stepper.ActionRow.Spacer />
+          <Button onClick={() => alert('Completed')}>Apply</Button>
+        </Stepper.ActionRow>
+      </div>
+    </Stepper>
+  )
+}
+```
+
+### In a modal
+
+A composition of a stepper with a `FullscreenModal`.
+
+```jsx live
+() => {
+  const steps = ['welcome', 'choose-cats', 'review'];
+  const [currentStep, setCurrentStep] = useState(steps[0]);
+  const [isOpen, open, close] = useToggle(false);
+  return (
+    <>
+      <Button variant="outline-primary" onClick={open}>Open fullscreen modal</Button>
+
+      <Stepper activeKey={currentStep}>
+        <FullscreenModal
+          title="My dialog"
+          className="bg-light-200"
+          isOpen={isOpen}
+          onClose={close}
+          beforeBodyNode={<Stepper.Header className="border-bottom border-light" />}
```

Comment:
What's the benefit of using beforeBodyNode and footerNode attributes rather than composing these components in between `ModalDialog.Header`, `ModalDialog.Body`, and `ModalDialog.Footer`?

Reply:
Minor. I wanted people to be able to leverage the defaults in fullscreen modal (that the header is dark) without having to reproduce them in the generic component. I'd be happy to not add these props in if you think having people use `ModalDialog` directly is the way to go.

Reply:
Nah, it's cool.  👍🏻

Reply:
Or, well, can you compose them instead if you want to?  Or are the props the only way to accomplish this?

Reply:
Yes you compose them if you wanted to.